.NET/Microsoft.Recognizers.Definitions.Common/English/TimeZoneDefinitions.cs

```diff
@@ -580,7 +576,6 @@ public static class TimeZoneDefinitions
             { @"pacific", -420 },
             { @"west coast", -420 },
             { @"west coast time", -420 },
-            { @"west coast timezone", -420 },

```

Comment:
Why remove some entries for "* timezone" in the minMapping dict and not the ones for "time" too?
It now look very strange were things are inconsistent. For example, with the new city names ("malaga" time vs. '
"zaragoza").